src/App.tsx

```diff
@@ -596,11 +593,20 @@ const Sidebar = (theme: Theme) => `
     right: 30px;
   }
 
+  /* slightly transparent fallback */
   .feedbot-wrapper {
     max-height: 100%;
-    background: linear-gradient(45deg, rgba(256,256,256, 0.2), rgba(256,256,256, 0.8));
-    backdrop-filter: blur(40px);
-    -webkit-backdrop-filter: blur(40px);
+    background: rgb(255, 255, 255);
+  }
+
+  /* if backdrop support: very transparent and blurred */
+  @supports ((-webkit-backdrop-filter: blur(40px)) or (backdrop-filter: blur(40px))) {
+    .feedbot-wrapper {
+      max-height: 100%;
+      background: linear-gradient(45deg, ${theme.mainColor}33, rgba(256,256,256, 0.8));
```

Comment:
What is the `33` constant here? And does it support `theme.mainColor` to be in both common color formats `rgb` and `hex`?

Reply:
The 33 after ```theme.mainColor``` is the opacity constant in HEX, it's used here so we don't have to covert hex to rgba. I can comment on it above to explain it, or come up with some converting logic.

Reply:
Oh, my silly mistake. Anyway `rgb` is used format so please wrap `theme.mainColor` by some function which will append `33` when string starting with `#` or append float opacity as the last argument when color in `rgb(x,y,z)` format.